src/util/config.js

```diff
@@ -17,7 +18,8 @@ const config: Config = {
         }
     },
     REQUIRE_ACCESS_TOKEN: true,
-    ACCESS_TOKEN: null
+    ACCESS_TOKEN: null,
+    MAX_PARALLEL_IMAGE_REQUESTS: 16
```

Comment:
Why default to 16?

Reply:
I had to pick a number anyway, and that's what [OpenLayers defaults to](https://github.com/openlayers/openlayers/blob/1035ba7bf2285ca5842505b5bdf5ec28ab62b64f/src/ol/source/Raster.js#L345) — it seemed to work well in practice, so I left it at that. We can adjust the default if you think 16 isn't a good value.